File: biner.py
def biner():
  while True:
    try:
      bilangan_str = input("Masukkan sebuah bilangan bulat (minimal 0): ")
      bilangan_int = int(bilangan_str)
      if bilangan_int < 0:
        print("Input tidak valid. Harap masukkan bilangan 0 atau lebih besar.")
        continue
      
      hasil = bin(bilangan_int)[2:]

      print(f"Angka biner dari {bilangan_int} adalah: {hasil}")
      break 

    except ValueError:
      print("Input tidak valid. Harap masukkan sebuah bilangan bulat.")
biner()

File: biner.dart
String cTB(int n) {
  return n.toRadixString(2);
}
void main() {
  int num1 = 7;
  print("Biner dari $num1 adalah: ${cTB(num1)}");

  int num2 = 5;
  print("Biner dari $num2 adalah: ${cTB(num2)}");

  int num3 = 3;
  print("Biner dari $num3 adalah: ${cTB(num3)}");

  int num4 = 6;
  print("Biner dari $num4 adalah: ${cTB(num4)}");
}